### ISSUE TITLE
I can't use the component as a directive in v-text-field

### ISSUE THREAD

**Issue by KammerR** [dm4t2/vue-currency-input#43]
Hello. I tried to use v-currency as a directive (following the tutorial strictly, I just changed the input to v-text-field), but it didn't work (every time I entered a value, the same back to 0)

**Comment by dm4t2** [comment]
Hi, which version of Vuetify/Vue Currency Input are you using? Example code would also be helpful.

Here is working example using the latest versions:

[![Edit Using Vue Currency Input with Vuetify](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/vue-template-kd7d1?fontsize=14&module=%2Fsrc%2FApp.vue)

**Comment by KammerR** [comment]
I am using the latest version of all. I'm using Nuxt, I put vue-currency-input as a plugin (although I tried to put it as a module, just like in the configuration). I created a codesandbox with my repo.

[https://codesandbox.io/s/github/KammerR/nuxtVcurrencyBuild/tree/master/basicNuxt](url)

**Comment by dm4t2** [comment]
It works if you remove the "v-currency-field" plugin from your Nuxt config (which also registers a `v-currency` directive globally):

https://codesandbox.io/s/basicnuxt-cr993

**Comment by npolyakoff** [comment]
it seems like with Vuetify 1.5.0 directive on v-text-field works not as in guide basic example. For instance, it possible to type more than 2 number after delimeter

**Comment by dm4t2** [comment]
@npolyakoff: I can not reproduce the issue you described. It also works fine with Vuetify 1.5.0.

**Comment by KammerR** [comment]
Thank you very much!!! it really worked ...

However, I found another problem (although I think this is a little bug)

When I use a v-slider to update the value, the field does not display the mask and resets to 0.

I think it's because V-text-field treats uses string and v-slider uses a number, but I didn't find a "right" way to fix it.

However, thanks for the help.

I made a update in repo to reproduce it:

https://codesandbox.io/s/github/KammerR/nuxtVcurrencyBuild/tree/master/basicNuxt
